### PR TITLE
Add logs for debug server down

### PR DESCRIPTION
**Pull request by Scorpiour**
Add access logger to inspect rest-api access
Adjust log4j format from text pattern to json with custom timestamp

### CODE REVIEW

**Review by david-sunw — APPROVED** [review]

File: hugegraph-api/src/main/java/com/baidu/hugegraph/api/profile/VersionAPI.java [hunk 1]
```diff
@@ -46,6 +46,7 @@ public Object list() {
                                        "core", CoreVersion.VERSION.toString(),
                                        "gremlin", CoreVersion.GREMLIN_VERSION,
                                        "api", ApiVersion.VERSION.toString());
+
```
Comment (zhoney) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/api/profile/VersionAPI.java]: delete empty line

File: hugegraph-core/src/main/java/com/baidu/hugegraph/StandardHugeGraph.java [hunk 2]
```diff
@@ -163,6 +163,8 @@
 
     private final RamTable ramtable;
 
+
```
Comment (zhoney) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/StandardHugeGraph.java]: ditto

File: hugegraph-core/src/main/java/com/baidu/hugegraph/task/TaskManager.java [hunk 3]
```diff
@@ -28,6 +29,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
+import com.baidu.hugegraph.util.*;
```
Comment (zhoney) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/task/TaskManager.java]: don't use wild pattern, every import class one line

File: hugegraph-api/src/main/java/com/baidu/hugegraph/api/filter/AccessLogFilter.java [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+    private static final Logger LOG = Log.logger(AccessLogFilter.class);
+
+    /**
+     * Use filter to log request info
+     * @param requestContext requestContext
+     * @param responseContext responseContext
+     */
+    @Override
+    public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext) throws IOException {
```
Comment (zhoney) [inline hugegraph-api/src/main/java/com/baidu/hugegraph/api/filter/AccessLogFilter.java]: improve align, we prefer not exceed 80 chars per line

**Review by zhoney — CHANGES_REQUESTED** [review]

**Review by zhoney — APPROVED** [review]

**Pull request merged**